java-sdk/src/main/java/org/sagebionetworks/bridge/sdk/models/upload/UploadFieldDefinition.java

```diff
@@ -102,6 +109,15 @@ public Integer getMaxLength() {
         return maxLength;
     }
 
+    /**
+     * Used for MULTI_CHOICE types. This lists all valid answers for this field. It is used by BridgeEX to create the
+     * Synapse table columns for MULTI_CHOICE fields. This is a list because order matters, in terms of Synapse
+     * column order. Must be specified if the field type is a MULTI_CHOICE.
```

Comment:
Worth noting that for surveys, this is the "server value" if it is specified, or else the label.